PROJECT_PROGRESSION.md

```diff
@@ -199,43 +199,51 @@ To graduate from At Large stage or for a new project to join as an Impact projec
 
 #### Examples
 
-1. Projects that are "complete" by the maintainers' standards.
-1. Projects that do not plan to release major versions in the future.
+1. Projects that have been accepted into incubation process after silent period.
 
 #### Expectations
 
-Projects in this stage are not in active development. Their maintainers may infrequently monitor their repositories, and may only push updates to address security issues, if at all. Emeritus projects should clearly state their status and what any user or contributor should expect in terms of response or support. If there is an alternative project the maintainers recommend, it should be listed as well. The foundation will continue to hold the IP and any trademarks and domains, but the project does not draw on foundation resources. 
+It is expected that incubating project will make an active effort to work through the on-boarding process in a reasonable timeframe. They will be offered foundation support to help do so.
 
 #### Acceptance Criteria
 
-Projects may be granted Emeritus status through CPC consensus and with the approval of the maintainers of the project.
-In cases where the project maintainers aren't responding after having been repeatedly contacted through appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
+1. consensus within private mailing list to move into incubation process
 
-### Incubating
+<!--If this checklist is updated please ensure `https://github.com/openjs-foundation/project-status/blob/HEAD/.github/ISSUE_TEMPLATE/02-project-sunsetting-checklist-template.md` is updated as well -->
+### End of Life
 
 #### Definition
 
-Incubating projects are projects that are in the process of completing their on-boarding checklist to join the foundation. Projects in this phase may be looking to join the foundation as At-Large or Impact Stage. Members of incubating projects are invited to join Cross Project Council meetings as a guest, please refer to the [CPC Charter](./CPC-CHARTER.md) for details on how guests may participate in the meetings.
+When an OpenJS software project reaches its end of life, it means that the developers responsible for it will no longer provide support, updates, patches, or enhancements for that version of the software. This includes security updates. The software doesn't disappear; it remains functional but won't receive any further official updates or fixes.
 
-#### Examples
+In most cases, End Of Life (EOL) projects will have a publicly communicated EOL milestone/date, indicating when they will no longer receive support or updates. Once this date is reached, the project should follow the steps outlined in the Archived project checklist.
```

Comment:
I am confused by this definition. The first paragraph says that EoL projects "will no longer provide support, updates, patches, or enhancements". But the second paragraph says that EoL projects are expected to have a milestone or date after which they'll be Archived, and _then_ "no longer receive support or updates".

Which one is it?

I think one core issue here may be the choice of terms, as "End of life" sounds really rather synonymous with "Archived" in this context. Could we not reconsider something like the current "Emeritus" term instead, which invokes retirement rather than death? Or maybe "Maintenance"?

From the further text, I get the sense that there's an expectation for this EoL phase to be a sort of reverse incubation to being "Archived", and for this process to be a somewhat active and directed one. So what's the plan for current emeritus projects like Sizzle and jQuery UI that have been [steadily](https://github.com/jquery/jquery-ui/commits/main/) receiving maintenance-y commits across the years? If/as they're being maintained, should they get bumped back up to "at-large"?

Reply:
This is a good point. I will reword to make this clearer. 

I think if projects are in maintenance mode and that they are regularly receiving security updates they should be bumped to active. To me, this is active development unless a date has been communicated when the development would stop. 

End Of Life is meant to communicate clearly. Hey, we are maintaining this now, but at a specified point in the future, we are going to stop.

Reply:
Reworked based on comments in this thread.